flexmeasures/ui/crud/accounts.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from flask import url_for
+from flask_classful import FlaskView
+from flexmeasures.ui.crud.api_wrapper import InternalApi
+from flexmeasures.ui.utils.view_utils import render_flexmeasures_template
+
+
+def get_accounts() -> list[dict]:
+    """/accounts"""
+    accounts = []
+    accounts_response = InternalApi().get(url_for("AccountAPI:index"))
+    accounts = accounts_response.json()
+
+    return accounts
+
+
+def get_account(account_id: str) -> dict:
+    account_response = InternalApi().get(url_for("AccountAPI:get", id=account_id))
+    account = account_response.json()
+
+    return account
+
+
+class AccountCrudUI(FlaskView):
```

Comment:
When opening `http://127.0.0.1:5000/accounts` while not logged in I get a `AttributeError: 'AnonymousUser' object has no attribute 'get_auth_token'`. I think there might be missing a authentication check on this `FlaskView`. I was expecting a redirect to the login page rather than a 500 status code.

Reply:
Specifically, the `@login_required` decorator. See `crud/assets.py` for an example. When opening `http://127.0.0.1:5000/assets` while not logged in you are redirected as expected.